src/screens/home/ducks/homeReducer.js

```diff
@@ -1,8 +1,25 @@
+import * as HomeActionTypes from "./home.types";
 
 const initialState = {
+    usersContacts: {}
 };
+
 export default function HomeReducer(state = initialState, action) {
     switch (action.type) {
+        case HomeActionTypes.ADD_CONTACT_SUCCESS:
+            let contact = action && action.data && action.data.name;
+            let contactsArr = [contact];
+            if(state.usersContacts[action.user]){
+                contactsArr = [...state.usersContacts[action.user], contact]
+            }
+            
+            return {
+                ...state,
```

Comment:
Vikas, this syntax is more cumbersome when we need to update a particular value inside a nested object. This can be used for small objects but not for large objects(like even if more than 2 or 3 levels deep). You can use a deep copy of this object here and update the copy and then return that.

Reply:
Thanks for the suggestion